tls/s2n_cipher_preferences.c

```diff
@@ -82,6 +82,45 @@ const struct s2n_cipher_preferences cipher_preferences_20210831 = {
     .suites = cipher_suites_20210831,
 };
 
+/*
+ * The following security policy is derived from the following specification:
+ * https://nvlpubs.nist.gov/nistpubs/SpecialPublications/NIST.SP.800-52r2.pdf
+ *
+ * Only supports TLS1.2
+ */
+struct s2n_cipher_suite *cipher_suites_default_fips[] = {
+    /* tls1.2 with ECDSA */
+    &s2n_ecdhe_ecdsa_with_aes_128_gcm_sha256,
+    &s2n_ecdhe_ecdsa_with_aes_256_gcm_sha384,
+    &s2n_ecdhe_ecdsa_with_aes_128_cbc_sha256,
+    &s2n_ecdhe_ecdsa_with_aes_256_cbc_sha384,
+
+    /* tls1.2 with RSA */
+    &s2n_ecdhe_rsa_with_aes_128_gcm_sha256,
+    &s2n_ecdhe_rsa_with_aes_256_gcm_sha384,
+    &s2n_ecdhe_rsa_with_aes_128_cbc_sha256,
+    &s2n_ecdhe_rsa_with_aes_256_cbc_sha384,
+    &s2n_dhe_rsa_with_aes_128_cbc_sha256,
+    &s2n_dhe_rsa_with_aes_256_cbc_sha256,
+    &s2n_dhe_rsa_with_aes_128_gcm_sha256,
+    &s2n_dhe_rsa_with_aes_256_gcm_sha384,
+
+    /* tls1.2, tls1.1, tls1.0 with ECDSA */
+    &s2n_ecdhe_ecdsa_with_aes_128_cbc_sha,
+    &s2n_ecdhe_ecdsa_with_aes_256_cbc_sha,
+
+    /* tls1.2, tls1.1, tls1.0 with RSA */
```

Comment:
Good callout. I also see the following. So I can double check that we want 1.1, 1.0 enabled. I can also update the comment which says `Only supports TLS1.2`

> Servers that support citizen or business-facing applications (i.e., the client may not be part of a
government IT system)10 shall be configured to negotiate TLS 1.2 and should be configured to
negotiate TLS 1.3. The use of TLS versions 1.1 and 1.0 is generally discouraged, but these
versions may be configured when necessary to enable interaction with citizens and businesses.

Reply:
we are dopping support for 1.1 and 1.0.

TLS1.2 is supported. TLS1.3 support is currently enabled but it will enable 1.3 protocol for existing customers so needs to be discussed